test/integration/narrow/routes/capture-debt.test.js

```diff
@@ -181,7 +181,7 @@ describe('capture-debt route', () => {
     expect(result.request.response.source.context.model.schemes).toStrictEqual(SCHEMES.map(scheme => scheme.name))
   })
 
-  test('POST /capture-debt with invalid scheme returns "The scheme must be one of the following: SFI, SFI Pilot, LNR, Vet Visits, Lump Sums." error message', async () => {
+  test('POST /capture-debt with invalid scheme returns "The scheme must be one of the following: SFI, SFI Pilot, CS, BPS, FDMR, Vet Visits, Lump Sums." error message', async () => {
```

Comment:
See this comment: https://github.com/DEFRA/ffc-pay-request-editor/pull/69/files#r1145027610